js/interactive-guides/modules/content-manager.js

```diff
@@ -479,12 +475,150 @@ var contentManager = (function() {
+
+    /** Reset the content in a specified FileEditor instance within a TabbedEditor
+     * @param {String} stepName - name of step where FileEditor is located
+     * @param {String} fileName - file name of editor within TabbedEditor
+     * @param {Integer} instanceNumber - (optional) zero-indexed instance number of FileEditor
+     */
+    var resetTabbedEditorContents = function(stepName, fileName, instanceNumber) {
+        var tabbedEditor = __getTabbedEditorInstance(stepName, instanceNumber);
+        if (tabbedEditor) {
+            var teditor = tabbedEditor.getEditorByFileName(fileName);
+            if (teditor) {
+                teditor.resetEditorContent();
+            }           
+        }        
+    };
+
+    /** Insert content before a certain line in a specified FileEditor instance
+     *  within a TabbedEditor.
+     * @param {String} stepName - name of step where TabbedEditor is located
+     * @param {String} fileName - file name of editor within TabbedEditor
+     * @param {Integer} lineNumber - line number to insert content above
+     * @param {String?} content - the content to put into the FileEditor
+     * @param {String} numberOfLines - (optional) number of lines in the new content; required only if inserting more than 1 line
+     * @param {Integer} instanceNumber - (optional) zero-indexed instance number of FileEditor
+     */
+    var insertTabbedEditorContents = function(stepName, fileName, lineNumber, content, numberOfLines, instanceNumber) {
+        var tabbedEditor = __getTabbedEditorInstance(stepName, instanceNumber);
+        if (tabbedEditor) {
+            var teditor = tabbedEditor.getEditorByFileName(fileName);
+            if (teditor) {
+                teditor.insertContent(lineNumber, content, numberOfLines);
+            }
+        }
+    };
+
+    /** Append content after a certain line in a specified FileEditor instance
+     *  within a TabbedEditor.
+     * @param {String} stepName - name of step where FileEditor is located
+     * @param {String} fileName - file name of editor within TabbedEditor
+     * @param {Integer} lineNumber - line number to append content below
+     * @param {String?} content - the content to put into the FileEditor
+     * @param {String} numberOfLines - (optional) number of lines in the new content; required only if appending more than 1 line.
+     * @param {Integer} instanceNumber - (optional) zero-indexed instance number of FileEditor
+     */
+    var appendTabbedEditorContents = function(stepName, fileName, lineNumber, content, numberOfLines, instanceNumber) {
+        var tabbedEditor = __getTabbedEditorInstance(stepName, instanceNumber);
+        if (tabbedEditor) {
+            var teditor = tabbedEditor.getEditorByFileName(fileName);
+            if (teditor) {
+                teditor.appendContent(lineNumber, content, numberOfLines);
+            }
+        }
+    };
+
+    /** Append content after a certain line in a specified FileEditor instance
```

Comment:
Sorry the javadoc in replaceEditorContents is wrong. It should be "Replace content ...". Could you fix the comments in both places, i.e. replaceTabbedEditorContents and replaceEditorContents.